### ISSUE TITLE
Slicing errror i get for some reason, sry to make an issue, dunno how to fix.

### ISSUE THREAD

**Issue by F1ks3r** [avocode/avocode#1156]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.11.0`
Source UID: 79145
```
TypeError: Cannot read property 'left' of undefined
    at declareAbsolutePosition (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/csshat-language-sass/main.js:104:33)
    at Sass.render (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/csshat-language-sass/main.js:222:11)
    at e.build (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:23:9426)
    at e._classRenderer (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:23:8252)
    at e.<anonymous> (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:23:6744)
    at e.render (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:23:8447)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:3:19881
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:246:17
    at /Applications/Avocode.app/Contents/Resources/
```

**Comment by vojtatranta** [comment]
Yes, we know about this bug will be fixed in `2.11.1` which we are releasing in 15 minutes.

**Comment by jakubzitny** [comment]
Fixed in `2.11.1`, closing.

Feel free to reopen if you have this or any other problem with Avocode 😉
